code/mission/missionparse.cpp

```diff
@@ -6396,7 +6396,10 @@ int mission_parse_get_multi_mission_info( const char *filename )
 }
 
 /**
- * Return the parse object on the ship arrival list associated with the given name
+ * Returns the parse object on the ship arrival list associated with the given name.
+ * Returns NULL if the object is not on the Arrival list, or if the object is a player
+ * ship that is already in game (Player ships never get removed from the arrival list
+ * in order to make respawns in multiplayer work).
  */
 p_object *mission_parse_get_arrival_ship(const char *name)
```

Comment:
Given touching this function and `mission_parse_get_arrival_ship()`, would suggest you provide the function documentation with Doxygen features. e.g. https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html